modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -74,6 +74,8 @@ This will print an output like:
 
 == Managing Extensions
 
+Extensions are microservices which can be started, stopped or instantinated. Because there is a difference between extensions which are part of the runtime and extensions outside the runtime, an in-depth documentation as been added to explain some background. Please read carefully to avoid unwanted behaviour. For details see the xref:deployment/extensions/extensions.adoc[Extensions] documentation.
```

Comment:
```suggestion
Extensions are microservices which can be started, stopped or instantiated. Because there is a difference between extensions which are part of the runtime and extensions outside the runtime, an in-depth documentation as been added to explain some background. Please read carefully to avoid unwanted behavior. For details see the xref:deployment/extensions/extensions.adoc[Extensions] documentation.
```